i18n/en/docusaurus-plugin-content-docs/current/homologation/overview.md

```
@@ -3,48 +3,44 @@ sidebar_position: 1
+Approval is the process of **validating and certifying** an application within the Nuvemshop ecosystem. 
+This process ensures that the app meets the expected technical and functional criteria, ensuring an efficient and secure integration.
+Depending on the type of application developed, the approval process may follow different paths, such as:
 
-- Submit the sequence diagram.
-- Send a demo video of the application.
-- Integration meeting (Only for payment, shipping, and ERP apps).
+* **Asynchronous Approval:** In this process, all artifacts must be submitted for review, and our team will perform validation tests. 
+* **Synchronous Approval:** This scenario applies to apps that handle sensitive data, such as ERP, Payments, and Shipping, where validation must be performed jointly. Our team will schedule a meeting with the requester to go through the app and validate it using a checklist.</br></br></br>
 
-## Requirements for Approving Integrated applications
+**Visibility and Next Steps**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)